src/helpers/post-helper.php

```diff
@@ -177,9 +177,9 @@ public function update_has_public_posts_on_attachments( $post_parent, $has_publi
 	 */
 	public function is_post_indexable( $post_id ) {
 		// Don't index posts which are not public (i.e. viewable).
-		$post_type    = \get_post_type( $post_id );
-		$public_types = $this->post_type->get_indexable_post_types();
-		if ( ! \in_array( $post_type, $public_types, true ) ) {
+		$post_type = \get_post_type( $post_id );
+
+		if ( ! $this->post_type->is_post_type_indexable( $post_type ) ) {
```

Comment:
I don't love the fact that we're creating a new method for this, that shares a name with an existing method (`WPSEO_Post_Type`'s `is_post_type_indexable()`). 

And the fact that the existing method is actually a wrapper for a method of the helper we're adding our new method makes this even weirder?

But then again, I don't see a better name for the new function, so maybe better to go away from this change and keep things as is?